build.gradle.kts

```diff
@@ -450,6 +457,22 @@ project(":js") {
     }
 }
 
+project(":ml-completion") {
+    intellij {
+        // BACKCOMPAT: 2020.2
+        if (platformVersion >= 203) {
+            setPlugins(mlCompletionPlugin)
+        }
```

Comment:
I suppose we also have to add dependency in `c-plugin` when we use `clion` as base IDE to avoid weird test fails because of `'navigation.class.hierarchy' is not found` error.
See `:toml` project as example

Reply:
Done